src/lib.rs

```diff
@@ -30,58 +30,134 @@
 //! The defaults provide a "full" engine experience, but you can easily enable / disable features
 //! in your project's `Cargo.toml` to meet your specific needs. See Bevy's `Cargo.toml` for a full list of features available.
 //!
-//! If you prefer it, you can also consume the individual bevy crates directly.
+//! If you prefer, you can also consume the individual bevy crates directly.
+//! Each module in the root of this crate, except for the prelude, can be found on crates.io
+//! with `bevy_` appended to the front, e.g. `app` -> [`bevy_app`](https://docs.rs/bevy_app/0.2.1/bevy_app/).
 
 #![doc(
     html_logo_url = "https://bevyengine.org/assets/icon.png",
     html_favicon_url = "https://bevyengine.org/assets/icon.png"
 )]
 
-mod default_plugins;
+/// `use bevy::prelude::*;` to import common components, bundles, and plugins.
 pub mod prelude;
 
-pub use bevy_app as app;
-pub use bevy_asset as asset;
-pub use bevy_core as core;
-pub use bevy_diagnostic as diagnostic;
-pub use bevy_ecs as ecs;
-pub use bevy_input as input;
-pub use bevy_math as math;
-pub use bevy_property as property;
-pub use bevy_scene as scene;
-pub use bevy_tasks as tasks;
-pub use bevy_transform as transform;
-pub use bevy_type_registry as type_registry;
-pub use bevy_utils as utils;
-pub use bevy_window as window;
+mod default_plugins;
 pub use default_plugins::*;
 
+pub mod app {
+    //! Build bevy apps, create plugins, and read events.
+    pub use bevy_app::*;
+}
+
+pub mod asset {
+    //! Load and store assets and resources for Apps
+    pub use bevy_asset::*;
+}
+
+pub mod core {
+    //! Contains core plugins and utilities for time.
+    pub use bevy_core::*;
+}
+
+pub mod diagnostic {
+    //! Useful diagnostic plugins and types for bevy apps.
+    pub use bevy_diagnostic::*;
+}
+
+pub mod ecs {
+    //! Bevy's entity-componenet-system.
```

Comment:
nit: `component`

Reply:
🤦

![screenshot 8984](https://user-images.githubusercontent.com/6868531/97648725-d8507700-1a2b-11eb-8247-73129155d692.png)

Reply:
I use that one too! I like it because it uses local dictionaries instead of communicating with a web service.